.gitignore

```diff
@@ -16,6 +16,7 @@ launch.json
 # datasets
 /src/store/_archive
 /src/store/sims
+/src/assets/_archive
```

Comment:
You can use a parent folder called _archive and gitignore that folder instead of adding each one.